Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -39,7 +39,7 @@ namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.Commands
         "ScriptAnalyzer",
         DefaultParameterSetName = "File",
         SupportsShouldProcess = true,
-        HelpUri = "http://go.microsoft.com/fwlink/?LinkId=525914")]
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkId=525914")]
```

Comment:
This link only re-directs to https://github.com/PowerShell/PSScriptAnalyzer/
Should we change it then to that?

Reply:
@joeyaiello opinion?

Reply:
the fwlink is fine - it's the way we handle links so it need not change if we move the repo. The real question is whether pointing to github is appropriate. I think for now it's fine.